pkg/kyc/component.go

```diff
@@ -663,3 +682,27 @@ func (c *component) sendSmsCodeGeneric(ctx context.Context, accessToken string,
 
 	return *smsCodeKc.Code, err
 }
+
+func (c *component) zipAttachments(attachments []apikyc.AttachmentRepresentation) ([]byte, error) {
+	buf := new(bytes.Buffer)
+
+	w := zip.NewWriter(buf)
+
+	for _, attachment := range attachments {
+		f, err := w.Create(*attachment.Filename)
```

Comment:
I suggest to set a creation date when adding files to avoid having an archive with files from 1970.

Reply:
Ok... I did not see that you were not using the same library as we did in papercard...
https://bitbucket.svc.elca.ch/projects/CLOUDTRUST/repos/2019.oc.paper-card-service/browse/internal/papercard/zipcreator.go#19
So may be archive/zip does not have this date issue